ambry-router/src/main/java/com.github.ambry.router/GetOperation.java

```diff
@@ -170,15 +170,16 @@ boolean isOperationComplete() {
 
   /**
    * Set the exception associated with this operation.
-   * A {@link ServerErrorCode#Blob_Deleted} or {@link ServerErrorCode#Blob_Expired} error overrides any other
-   * previously received exception.
+   * A {@link ServerErrorCode#Blob_Deleted} or {@link ServerErrorCode#Blob_Expired} or {@link ServerErrorCode#Blob_Expired}
```

Comment:
minor, the new one should be `Blob_Authorization_Failure`?

Reply:
fixed.